docs/index.rst

```diff
@@ -268,6 +268,35 @@ then the *hookimpl* should be marked with the ``"optionalhook"`` option:
 
         return config
 
+.. _specname:
+
+Hook implementation naming
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+During plugin :ref:`registration <registration>`, hook implementations are
+matched to a hook :ref:`specifications <specs>` in the *host* program with the
+same name as the function being decorated by ``@hookimpl`` (e.g.
+``setup_project`` in the example above).
+
+*new in version 0.13.2:*
```

Comment:
obviously, took a guess at this... feel free to directly edit this (or any other part of this PR).